### PR TITLE
Google Analytics and Defer Assets

### PR DESCRIPTION
**Pull request by fws-dusan**
-Added Google Analytics ACF and implemented it in the header. -Implemented "current template" script into Defer Assets

https://app.clickup.com/t/32bffke

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -9,6 +9,8 @@
  * @package fws_starter_s
  */
 
+$google_analytics_tag_code = get_field( 'google_analytics_tag_code', 'options' );
```
Comment (nikolatopalovic) [inline header.php:12]: This should be placed in `if` conditional statement.

File: header.php [hunk 2]
```diff
@@ -23,6 +25,19 @@
 	<link rel="preload" href="<?php echo get_template_directory_uri(); ?>/src/assets/fonts/OpenSans-Bold.woff2" as="font" crossorigin />
 	<link rel="preload" href="<?php echo get_template_directory_uri(); ?>/src/assets/fonts/OpenSans-Regular.woff2" as="font" crossorigin />
 
+	<!-- Global site tag (gtag.js) - Google Analytics -->
```
Comment (nikolatopalovic) [inline header.php:28]: Same here... we cannot render this if the field is empty.

**Review by nikolatopalovic — CHANGES_REQUESTED** [review]

**Review by nikolatopalovic — APPROVED** [review]

File: header.php (hunk 1 above)
Comment (fws-bojan-kolano) [reply header.php]: @nikolatopalovic This is what we needed if I understood correctly. `$google_analytics_tag_code = get_field( 'google_analytics_tag_code', 'options' ) ?? '';`

File: header.php (hunk 2 above)
Comment (fws-bojan-kolano) [reply header.php:28]: @nikolatopalovic I wrapped the whole GA in the if statement as instructed `
	
	<?php if($google_analytics_tag_code): ?>
		<script async src="https://www.googletagmanager.com/gtag/js?id=<?php echo $google_analytics_tag_code ?>"></script>
		<script>
			window.dataLayer = window.dataLayer || [];
			function gtag() {
				dataLayer.push(arguments);
			}
			gtag('js', new Date());

			gtag('config', '<?php echo $google_analytics_tag_code ?>');
		</script>
	<?php endif; ?>`

**Pull request merged**